src/main/resources/assets/viafabric/lang/pt_br.json

```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "Habilitar ViaVersion",
   "gui.enable_client_side.question": "Você tem certeza de que deseja habilitar o modo client-side?",
   "gui.enable_client_side.warning": "Não posso garantir que este mod seja permitido em todos os (ou mesmo em quaisquer) servidores. Esse poderá causar problemas com plugins anti-trapaça. USE POR SUA PRÓPRIA CONTA E RISCO.",
-  "gui.enable_client_side.enable": "Habilitar"
+  "gui.enable_client_side.enable": "Habilitar",
+  "gui.viafabric_config.title": "Configurações de ViaFabric",
+  "gui.client_side.enable": "Habilitar o lado do cliente",
+  "gui.client_side.disable": "Desactivar o lado do cliente",
```

Comment:
"Desactivar" is pt-pt

Reply:
It's actually both Spanish and Portuguese

Reply:
I meant that "desactivar" isn't used in brazilian portuguese, it is usually used in european portuguese

Reply:
okay i changed it anyways